docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -19,22 +19,28 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/deployment/add-semgrep-to-ci).
 
-There are two features provided by Semgrep to split up a repo. Consider a monorepo named `monorepo` with four main modules
+There are two features provided by Semgrep to split up a repo. Consider a monorepo named `monorepo` with four main modules:
 
-    /src/moduleA
-    /src/moduleB
-    /src/moduleC
-    /src/moduleD
+ ```
+/src/moduleA
+/src/moduleB
+/src/moduleC
+/src/moduleD
+```
 
-The easiest way to split this monorepo up is into four separate scans, one for each module. To do this, use the `--subdir` (see `semgrep ci --help`) flag with the relevant path to only scan files in that module's code path:
+The easiest way to split this monorepo up is into four separate scans, one for each module. To do this, use the `--subdir` flag with the relevant path to only scan files in that module's code path:
```

Comment:
I didn't feel like the reference added very much here and it felt awkward with the flow, but open to feedback on another place to put it, or why it is worthwhile.